runtime/compiler/env/J9SharedCache.hpp

```diff
@@ -123,6 +123,15 @@ class TR_J9SharedCache : public TR_SharedCache
     */
    virtual bool isPointerInSharedCache(void *ptr, uintptrj_t *cacheOffset = NULL);
 
+   /**
+    * \brief Checks whether the specified offset is within the shared cache.
+    *
+    * \param[in] offset The offset to check.
+    * \param[out] ptr If offset is within the shared cache and this parameter is not NULL the result of converting offset into a pointer will be returned here. If offset is not within the shared cache this parameter is ignored.
+    * \return True if the offset is within the shared cache, false otherwise.
+    */
+   virtual bool isOffsetInSharedCache(uintptrj_t offset, void *ptr = NULL);
```

Comment:
This would be clearer and closer to the truth, if the second parameter type was `void **`, or `uintptrj_t *`.